xrpl4j-address-codec/src/test/java/org/xrpl/xrpl4j/codec/addresses/AddressTest.java

```diff
@@ -0,0 +1,52 @@
+package org.xrpl.xrpl4j.codec.addresses;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.Test;
+import org.xrpl.xrpl4j.model.transactions.Address;
+
+public class AddressTest {
+
+  private final AddressCodec addressCodec = new AddressCodec();
```

Comment:
Can remove this.

Reply:
Got added back, since I moved the checksum test back here!